auth-web/src/components/auth/staff/continuation-application/HomeJurisdictionInformation.vue

```diff
@@ -52,15 +52,15 @@
       </v-row>
     </section>
 
-    <!-- Identifying Number in Home Jurisdiction -->
+    <!-- Identifying Number in Previous Jurisdiction -->
```

Comment:
Please match comment to code.

Also on lines 77 and 121.

Reply:
Updated